src/app/core/services/helper.service.ts

```diff
@@ -115,7 +115,7 @@ export class HelperService {
     }
   }
 
-  getOffset(el: any) {
+  getOffset(el: Element) {
```

Comment:
will eventually strongly type, but I will revert this now as there are dc.js charts using this.